k8s/values.yaml

```diff
@@ -0,0 +1,30 @@
+deploy_version: 0.1.12
+image_tag: latest
+
+applications:
+  - name: flask
+    image: pennlabs/labs-api-server
+    secret: labs-api-server
+    ingress:
+      hosts:
+        - host: api.pennlabs.org
+          paths: ["/"]
+    secretMounts:
+      - name: ios_key
+        item: ios_key
+        path: "/app/ios_key.p8"
+  - name: redis
+    image: redis
+    tag: 6.0rc
+    port: 6379
+cronjobs:
+  - name: laundry
+    schedule: "*/15 * * * *"
+    secret: labs-api-server
+    image: pennlabs/labs-api-server:latest
```

Comment:
remove latest from these and bump icarus version to .14

Reply:
Fixed